share/shwap/eds_id.go

```diff
@@ -0,0 +1,68 @@
+package shwap
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/celestiaorg/celestia-node/share"
+)
+
+// EdsIDSize defines the byte size of the EdsID.
+const EdsIDSize = 8
+
+// EdsID represents a unique identifier for a row, using the height of the block
+// to identify the data square in the chain.
+type EdsID struct {
```

Comment:
I'm wondering why we need it as a separate type and not a part of the RowID?

Reply:
So we can request EDSes